go.mod

```diff
@@ -24,6 +23,7 @@ require (
 	github.com/gorilla/mux v1.8.1
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.11.3
+	github.com/imua-xyz/price-feeder v0.1.16-0.20250219142917-5ea492552d53
```

Comment:
use v0.2.3?

Reply:
Not yet; it has not been rebranded and causes `Go` to complain about the mismatch in `go.mod` of this project and the imported project (or further deeper errors if I try to work around it). Instead using v0.1.15 rebranded temporarily https://github.com/imua-xyz/price-feeder/tree/rebrand.